engine/card_flip.asm

```diff
@@ -2,6 +2,13 @@ CARDFLIP_LIGHT_OFF EQU $ef
 CARDFLIP_LIGHT_ON  EQU $f5
 CARDFLIP_DECK_SIZE EQU 4 * 6
 
+; two labels below called from inside ./dummy_game.asm
+Unknown_e00ed: ; e00ed (38:40ed)
+; Graphics for an unused Game Corner
+; game were meant to be here.
+Ret_e00ed: ; e00ed (38:40ed)
```

Comment:
What about `Stubbed_` or `Dummy_` here?

Reply:
Also, the name of `Unknown_e00ed` could reflect that it was some 4-tile image for the dummy game. (Most likely a cursor, given that `SPRITE_ANIM_INDEX_DUMMY_GAME` uses 4 tiles at $00 just like other cursors.)

Reply:
I left empty functions with generic names, and only used meaningful names when there is actual code (even if unreferenced or dummied out after a ret) to back it up. If this one's so evident though, I guess it could be changed to something to reflect what it was supposed to be there, but I feel it's enough if reflected in a comment.